### DISCUSSION TITLE
🆕 Chromatic visual testing integration for Cypress

### DISCUSSION THREAD

**Discussion by joevaugh4n** [cypress-io/cypress#28482] [General]
Hey there, Cypress community! I'm one of the maintainers of Storybook and a developer at Chromatic. 

We've just shared the first-look at Chromatic's first-ever visual testing integration for Cypress E2E tests.

https://www.chromatic.com/blog/introducing-cypress-integration-chromatic/

If anyone's interested in trying it out, we'd love your feedback to make the integration better! There's a Google Form for early access signups at the bottom of the article 🥰

Thank you!

**Comment by baluanne9999** [comment]
Good Morning, Hope you are doing great. I did followed the exact same steps that has been mentioned in the document . However I am not seeing any screenshots in the Chromatic. Can you kindly suggest if there is anything that I am missing

**Comment by balaarunreddyv1** [comment]
Yes this implementation doesn't work.
